en/docs/administer/monitoring-logs.md

```diff
@@ -59,39 +55,45 @@ See the following information on configuring **Carbon logs**, **Audit
 logs,** **HTTP access logs**, and **Service/Event logs** for WSO2 Identity Server.
 
 #### Configuring Carbon logs 
-You can easily configure Carbon logs using the WSO2 Identity Server Management Console or you can manually edit the
+Carbon logs are enabled in WSO2 Identity Server by default. You can change
+the following default configuration by manually updating the the
 `           log4j2.properties          ` file. 
 
+```toml
+
+# Appender config to CARBON_LOGFILE
+appender.CARBON_LOGFILE.type = RollingFile
+appender.CARBON_LOGFILE.name = CARBON_LOGFILE
+appender.CARBON_LOGFILE.fileName = ${sys:carbon.home}/repository/logs/wso2carbon.log
+appender.CARBON_LOGFILE.filePattern = ${sys:carbon.home}/repository/logs/wso2carbon-%d{MM-dd-yyyy}.log
+appender.CARBON_LOGFILE.layout.type = PatternLayout
+appender.CARBON_LOGFILE.layout.pattern = TID: [%tenantId] [%appName] [%d] [%X{Correlation-ID}] %5p {%c} - %mm%ex%n
+appender.CARBON_LOGFILE.policies.type = Policies
+appender.CARBON_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
+appender.CARBON_LOGFILE.policies.time.interval = 1
+appender.CARBON_LOGFILE.policies.time.modulate = true
+appender.CARBON_LOGFILE.strategy.type = DefaultRolloverStrategy
+appender.CARBON_LOGFILE.strategy.max = 20
+appender.CARBON_LOGFILE.filter.threshold.type = ThresholdFilter
+appender.CARBON_LOGFILE.filter.threshold.level = DEBUG
+```
+
+The log levels that
+can be configured are [listed below](#MonitoringLogs-log4j-levels).
+
+!!! note
+    All the changes that are made to the log4j2.properties will be apply at run time.
```

Comment:
please change "will be apply" to "will apply" or "will be **applied**"

Reply:
done